static/src/js/components/Tour/TourSteps.jsx

```diff
@@ -642,7 +642,7 @@ const TourSteps = ({
               <ExternalLink
                 href="https://www.earthdata.nasa.gov/learn/earthdata-search"
               >
-                Earthdata Search wiki
+                Earthdata Search
```

Comment:
Some discussion with PO around this; since this link is not actually a wiki page this language change is preferred